docker-app/qfieldcloud/core/models.py

```diff
@@ -1186,21 +1186,32 @@ def problems(self) -> list[dict]:
                 layer_name = layer_data.get("name")
 
                 if layer_data.get("error_code") != "no_error":
+                    if layer_data.get("error_code") == "localized_dataprovider":
+                        description = _('Layer "{}" dataprovider is localized').format(
+                            layer_name
+                        )
+                        solution = _(
+                            'Make sure your <a href="https://docs.qfield.org/fr/how-to/outside-layers/">localized layer</a> is available on your QField device.'
```

Comment:
not sure whether we should add the 

```
from django.utils.safestring import mark_safe

...

mark_safe(_(...))
```

Reply:
I had no idea what will happen if we pass the returning object to `json` serializer. All is good:

```
>>> a = mark_safe("<span>hello</span>")
>>> type(a)
<class 'django.utils.safestring.SafeString'>
>>> import json
>>> json.dumps(a)
'"<span>hello</span>"'
```